test/unit/gateways/hps_test.rb

```diff
@@ -142,6 +142,20 @@ def test_failed_refund
     assert_failure refund
   end
 
+  def test_successful_credit
```

Comment:
The majority of the other tests utilize `assert_instance_of` before the `assert_failure` or `assert_success` lines. Would adding that line benefit these new tests?

Reply:
I saw that and I didn't see any reason for it. It's just asserting that the class is a `Response` object, which is going to always be true.